docker/Dockerfile

```diff
@@ -105,7 +106,14 @@ RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest
 
 # Add code stub last
 COPY . /root/code/garage
+# Prevent garage from exiting on config
+RUN cp /root/code/garage/garage/config_personal_template.py \
+  /root/code/garage/garage/config_personal.py
 RUN ["/bin/bash", "-c", "source activate garage && pip install -e /root/code/garage && source deactivate garage"]
 
+# Specifies which docker image flavor we're dealing with
+ARG IMAGE_TYPE
```

Comment:
this is strange. why does the Dockerfile need to know this?

Reply:
I realized that the docker-entrypoint for headless and nvidia shares the same commands except for creating the socket of the fake display for the headless type.
In order to distinguish between both I added this variable and I use it [here](https://github.com/rlworkgroup/garage/blob/b97856de130d0b5eebdfeb400796dc1eee458a78/docker/docker-entrypoint.sh#L8). My effort was to keep one canonical entry point for all image types, but now I see that the idea is that each type should have its own Dockerfile and entrypoint.